conf/default.cfg

```diff
@@ -118,7 +118,7 @@ lang = en
 
 #options for testing the word similarity module
 [word_sim]
-4langpath = /home/recski/sandbox/4lang
+4langpath = /home/eszter/projects/4lang
```

Comment:
This line should not be part of the commit - it'll cause modules to stop working on our other servers. The environment variable FOURLANGPATH will be a better solution, of course...